packages/expandable-card/src/ExpandableCard/ExpandableCard.tsx

```diff
@@ -53,6 +53,8 @@ const ExpandableCard = ({
   const summaryId = useIdAllocator({ prefix: 'expandable-card-summary' });
   const contentId = useIdAllocator({ prefix: 'expandable-card-content' });
 
+  const transitionRef = useRef<null>(null);
```

Comment:
`<null | HTMLElement>`?

Reply:
Unless things changed in R18, typing this as `HTMLElement` should be fine as React will recognize `null` as a valid type if `useRef<HTMLElement>`